projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -13,6 +13,41 @@
  * Class Jetpack_Admin_Menu.
  */
 class Jetpack_Admin_Menu extends Admin_Menu {
+
+	/**
+	 * Returns an array with the CPT menu items different form post and page types.
+	 *
+	 * @param array $menu The menu array to filter from.
+	 * @return array The filtered menu array.
+	 */
+	private function filter_cpt_menu_items( $menu ) {
+		$exclude_slugs = array( 'menu-posts', 'menu-pages' );
```

Comment:
Non-blocking question: Why don't we need to exclude `menu-feedback` here? That's another CPT so I expected the Feedback menu to be duplicated, but it is not for some reason I don't understand.

Reply:
The `menu-feedback` has the `show_in_menu` equal to false, and that's why it doesn't appear by default on the global variable `$menu`.

https://github.com/Automattic/jetpack/blob/765e0cea4851ae116b59a0408b312ebbd90b01ce/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php#L326

Reply:
Nice finding! I guess we want to show the menu in Calypso but not in WP Admin, so that explains why we still need the `add_feedback_menu` exception.